hack/allure-report.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -ex
+
+CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-docker}
+
+$CONTAINER_RUNTIME info 2>/dev/null >/dev/null || NO_CONTAINER_RUNTIME=$?
+if [ -n "$NO_CONTAINER_RUNTIME" ]; then
+    echo "WARNING $CONTAINER_RUNTIME is required but not installed, skipping..."
+    exit 1
+fi
+
+ACTION=${1:-generate}
+
+NAME=test-acceptance-report
+
+REPORT_DIR=$(pwd)/out/acceptance-tests-report
+mkdir -p $REPORT_DIR
+
+$CONTAINER_RUNTIME build -f test/acceptance/Dockerfile.allure -t $NAME test/acceptance
+
+if [ $($CONTAINER_RUNTIME ps -a -f name=$NAME | grep $NAME | wc -l) != 0 ]; then $CONTAINER_RUNTIME rm -f $NAME; fi
```

Comment:
should not we stop container only if action is `generate`?

Reply:
We are running a container with the same `$NAME` in case of both actions. The point of the step is to ensure that there is no such container running when we start it (in both actions).

Reply:
+1 on simplifying it